src/plugins/kibana/index.js

```diff
@@ -18,14 +18,14 @@ module.exports = function (kibana) {
         title: 'Kibana',
         listed: false,
         description: 'the kibana you know and love',
-        //icon: 'plugins/kibana/settings/sections/about/barcode.svg',
```

Comment:
Odd, this is still showing up. It's been removed in the file: https://github.com/tylersmalley/kibana/blob/8439192cdc7f5ea27f769d66d3a239d2eb871b20/src/plugins/kibana/index.js